druid/src/text/selection.rs

```diff
@@ -28,13 +28,20 @@ pub struct Selection {
 
     /// The active edge of a selection, as a byte offset.
     pub end: usize,
+
+    /// The saved horizontal position, during vertical movement.
+    pub h_pos: Option<f64>,
```

Comment:
It's a little weird how this variable is called `h_pos` here and `x_pos` on 105

Reply:
I agree :)